src/components/Form/SearchWrapper.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import T from 'prop-types'
+import { randomId, classNames } from '../../utils/'
+
+const initial = 'c-search'
+
+const SearchWrapper = ({ id, className, submitPosition, ...rest }) => (
+  <form
+    id={randomId()}
+    role="search"
+    action=""
```

Comment:
action and method should be passed in via props.

Reply:
done